### ISSUE TITLE
"Disabled" option on checkboxes

### ISSUE THREAD

**Issue by craigjh** [18F/web-design-standards#338]
In the Checkbox sample, the Disabled option is shown with a solid filled color (black?) to designate it is disabled.  If a user has four choices, like the example, but WITHOUT a pre-checked option, IMO the disabled option is going to appear to be a pre-selected choice, not a disabled choice. This could be confusing or lead to errors for users in a hurry.

**Comment by maya** [comment]
@craigjh great point as well!

How about if we change the text to an accessible but lighter gray and go lighter with the fill on the checkbox too?

Current:
![screen shot 2015-08-24 at 6 03 01 pm](https://cloud.githubusercontent.com/assets/5249443/9456293/72101d86-4a8a-11e5-9e70-12e3ba972d24.png)

Light gray:
![screen shot 2015-08-24 at 6 01 54 pm](https://cloud.githubusercontent.com/assets/5249443/9456296/7ab77efc-4a8a-11e5-9e96-443a6c54a17d.png)

Lighter gray:
![screen shot 2015-08-24 at 6 01 44 pm](https://cloud.githubusercontent.com/assets/5249443/9456301/7f18b0ba-4a8a-11e5-9dad-90e2ae961ac3.png)

**Comment by carodew** [comment]
Perhaps the disabled checkbox should not have a fill at all?

**Comment by mollieru** [comment]
cc @angel

**Comment by angel** [comment]
@craigjh @maya @carodew @mollieru Here is the v2 version of the checkboxes. Would our gray-lightest work?
![inputs_checkboxes_ui_v2](https://cloud.githubusercontent.com/assets/103496/9508168/779409c8-4c21-11e5-82a6-0f75811aec16.png)

**Comment by maya** [comment]
This was already done.
